presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestTypedSet.java

```diff
@@ -130,9 +134,29 @@ public void testBigintSimpleTypedSet()
         }
     }
 
+    @Test
+    public void testMemoryExceeded()
```

Comment:
something is fishy, a single large element does not seem to contribute to teh elementBlock.sizeInBytes()

i'll dig more why
because I can java OOM without triggering the expected exception using a single large array element vs an array of many elements